.circleci/config.yml

```diff
@@ -121,6 +158,29 @@ workflows:
       - go1.11-alpine:
           requires:
             - get_source
-      - singularity_test:
+      - unit_test:
           requires:
             - get_source
+  daily_CI:
+    triggers:
+      - schedule:
+          cron: "0 0 * * *"
```

Comment:
I'd suggest running this workflow on each commit to `master` and `release` branches. When this fails on a nightly run, it may not be obvious which PR broke things if more than one PR was merged that day. Also, the quicker this feedback is provided to the developer responsible for the change, the better. Feedback will be tens of minutes after the PR is merged, rather than up to 24 hours.